spring-cloud-aws-ses/src/main/java/io/awspring/cloud/ses/SimpleEmailServiceMailSender.java

```diff
@@ -125,7 +125,9 @@ protected String getConfigurationSetName() {
 	private SendEmailRequest prepareMessage(SimpleMailMessage simpleMailMessage) {
 		Assert.notNull(simpleMailMessage, "simpleMailMessage are required");
 		Destination.Builder destinationBuilder = Destination.builder();
-		destinationBuilder.toAddresses(simpleMailMessage.getTo());
+		if (simpleMailMessage.getTo() != null) {
```

Comment:
does there need to be a check that at least one of either to, CC or BCC have been filled out?